packages/react-components/react-badge/src/components/PresenceBadge/usePresenceBadgeStyles.styles.ts

```diff
@@ -71,32 +71,50 @@ const useStyles = makeStyles({
     color: tokens.colorNeutralForeground3,
   },
 
-  // Icons are not resizeable, and these sizes are currently missing
-  // use `!important` to size the currently available icons to the missing ones
-  //
   tiny: {
     aspectRatio: '1',
     width: '6px',
     backgroundClip: 'unset', // tiny icons have a border less than 1px wide, and can't use the backgroundClip fix
     '& svg': {
-      width: '6px !important',
-      height: '6px !important',
+      fontSize: '6px',
+    },
+  },
+  'extra-small': {
+    aspectRatio: '1',
+    width: '10px',
+    backgroundClip: 'unset', // tiny icons have a border less than 1px wide, and can't use the backgroundClip fix
+    '& svg': {
+      fontSize: '10px',
+    },
+  },
+  small: {
+    aspectRatio: '1',
+    width: '12px',
+    backgroundClip: 'unset', // tiny icons have a border less than 1px wide, and can't use the backgroundClip fix
+    '& svg': {
+      fontSize: '12px',
+    },
```

Comment:
`fontSize` should work without the `'& svg'` selector. And in fact, I think you should be able to remove the `aspectRatio` and `width` as well, now that it's using resizable icons that respond to `fontSize`:
```suggestion
    fontSize: '12px',
```

Reply:
Ditto for all other sizes as well.